package.json

```
@@ -31,9 +31,17 @@
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
```

We have updated the dependencies on the master branch and should make sure yours match before merging
```suggestion
    "react-dom": "^16.5.2",
```
```suggestion
    "react-dom": "^16.5.2",
```
```suggestion
    "react-dom": "^16.5.2",
```
```suggestion
    "react-dom": "^16.5.2",
```